pages.bn/android/settings.md

```diff
@@ -0,0 +1,20 @@
+# settings
+
+> Android OS সম্পর্কে তথ্য পান।
+> আরও তথ্য পাবেন: <https://adbinstaller.com/commands/adb-shell-settings-5b670d5ee7958178a2955536>.
+
+- `global` নামস্থানে সেটিংসের একটি তালিকা প্রদর্শন করুন:
+
+`settings list {{গ্লোবাল}}`
+
+- একটি নির্দিষ্ট সেটিং এর মান পান:
+
+`settings get {{global}} {{airplane_mode_on}}`
+
+- একটি সেটিং এর মান সেট করুন:
+
+`settings put {{system}} {{screen_brightness}} {{42}}`
```

Comment:
```suggestion
- একটি সেটিং এর একটি নির্দিষ্ট মান সেট করুন:

`settings put {{system}} {{screen_brightness}} {{0..100}}`
```

Almost the same issue as above.
Note: `Set a specific value of a setting:` can be used as a translation source.
Optional fix (error source: original page): unexpected `42` integer, `0..100` expected.